src/pages/vendor/VendorAddProductPage/ProductDescriptionForm/index.tsx

```diff
@@ -323,12 +341,27 @@ function ProductDescriptionForm() {
                 const selectedOptions = selected as colorOptions[];
 
                 if (selectedOptions.length === 0) {
-                  return <Typography>{colors[0].value}</Typography>;
+                  return (
+                    <Typography>
+                      {t('editProduct.colorsPlaceholder')}
+                    </Typography>
+                  );
                 }
 
                 return selectedOptions.join(', ');
               }}
-              onChange={(v) => setProductColors(v.target.value as string[])}
+              onChange={(v) => {
+                const selectedValues = v.target.value as string[];
+
+                setProductColors(selectedValues);
+
+                if (selectedValues.length >= 1) {
+                  setColorsSelectOpen(false);
```

Comment:
you can write this expression like if(selectedValues.length) without >=1

Reply:
I cannot write like this, because I got a syntax error